README.md

```diff
@@ -74,10 +74,77 @@ If you want to add your theme:
 
     ...
     ```
+*   Commit only one message, more details in the **Commit Message**
```

Comment:
Do you mean: commit only once per PR? In that case, I would say that you can still have more than one commit, as long as they respect the other commit conventions.

Reply:
When creating a PR, make sure that there is only one commit message, which can reduce the number of the commit messages.

This means that when you commit the message, all parts of the theme or scheme are ready.

You can commit other messages after the PR is created. Maybe the contributor fix a bug and commit it, or the collaborator request further changes. This is impossible to avoid.

Reply:
Then I would say something like "Commit only once, more details in the **Commit Message**", which I find easier to understand

Reply:
Yes, "once" is easier to understand.